build/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:bullseye-slim
+
+ENV AWS_DEFAULT_REGION=eu-west-1
```

Comment:
I am unable to figure out why this is necessary :upside_down_face: Perhaps you can enlighten me? :bow:

Reply:
By the way, I can conclude that it is indeed necessary.

Reply:
I have moved it from the Dockerfile to the spec_helper.rb in https://github.com/clearhaus/dynamodb-mutex/pull/7/commits/1dfba8ffed8c07415481dd98c9885da1493016cf

I'm not sure but think it is because this expects it to be set https://github.com/clearhaus/dynamodb-mutex/blob/1dfba8ffed8c07415481dd98c9885da1493016cf/spec/spec_helper.rb#L56